barbican/cmd/barbican_manage.py

```diff
@@ -344,10 +347,38 @@ def _verify_label_does_not_exist(self, key_type, label, session):
             print("The label {label} already exists!".format(label=label))
             sys.exit(1)
 
+class SAPCommands(object):
+
+    description = "Move all secrets associated with the old_project_id to the new_project_id."
+
+    @args('--db-url', '-d', metavar='<db-url>', dest='dburl',
+          help='Barbican database URL')
+    @args('--old-project-id', '-o', metavar='<old-project-id>', dest='old_project_id',
+          help='The old project ID to move secrets from.')
+    @args('--new-project-id', '-n', metavar='<new-project-id>', dest='new_project_id',
+          help='The new project ID to move secrets to.')
```

Comment:
These two args should also have `required=True` as the command cannot work without them.